README.md

```diff
@@ -9,15 +9,76 @@ Here is an example of a Grafana dashboard for an ESP project. This dashboard rel
 
 <img alt="Sailing dashboard" src="img/sailing-dashboard.png"  width="50%" height="50%">
 
-## Prerequisites
+
+## Getting Started
+The following steps provide an example of how to get started with the plug-in. 
+
+### Prerequisites
+Have a running ESP Marketplace deploy.
```

Comment:
There are 2 prereqs here so how about a bulleted list, as shown below. Notice also how I have referred to Marketplace as "SAS Event Stream Processing in the Microsoft Azure Marketplace":

- A running deployment of SAS Event Stream Processing in the Microsoft Azure Marketplace.
- An ESP project running in either SAS Event Stream Processing Studio or SAS Event Stream Manager.

Reply:
Done